data/io.snapcraft.SnapDesktopIntegration.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Icon=snapd-desktop-integration.svg
-Name=Snapd Desktop Integration
+Name=Snaps
```

Comment:
Is this shown somewhere?

Reply:
BTW: now that I look at it again, I see that I didn't implement showing the updates when clicking the notification...